Sources/Workspace/InitPackage.swift

```diff
@@ -469,7 +471,7 @@ extension SupportedPlatform {
             guard self.version.patch == 0 else {
                 return false
             }
-        } else if [Platform.macOS, .iOS, .watchOS, .tvOS].contains(platform) {
+        } else if [Platform.macOS, .iOS, .watchOS, .tvOS, .driverKit].contains(platform) {
             guard self.version.minor == 0, self.version.patch == 0 else {
                 return false
             }
```

Comment:
A few lines down needs to add:

    case .driverKit:
        return (19...20).contains(version.major)